stats/DataAggregation.md

```diff
@@ -13,6 +13,10 @@ The library SHOULD provide support for multiple types of Aggregations:
 * `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
 * `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
 mean value.
+* `Max`: indicates that data collected and aggregated with this `Aggregation` will calculate the 
```

Comment:
Stackdiver Monitoring has a concept [Range](https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#Range), which includes both max and min values. I think using a similar `Range` aggregation probably makes more sense.